src/components/Match/MatchStory.jsx

```diff
@@ -156,6 +156,32 @@ const formatList = (items, noneValue = []) => {
   }
 };
 
+// evaluate the sentiment behind the message - rage, question, statement etc
+const evaluateSentiment = (event, lastMessage) => {
+  const { message, player, time } = event;
+  const sentiment = message.indexOf('?') !== -1 ? ['question'] : ['statement'];
```

Comment:
Small note: this'll render any message with a `?` anywhere as a question, no? I think this sounds reasonable (even though we can get messages like 
`"que? you suck m8", Ember Spirit asked`
tagged with "asked"), but worthwhile to point out explicitly. Maybe a future PR could break up sentence fragments so we can split text like the above into something like
`"que?" Ember Spirit asked, "you suck m8"`.  :+1:

Reply:
Breaking up sentence fragments sounds fantastic! Would require a bit of a refactor around ChatMessageEvent. I'll have a look into it, but definitely not in this PR. Don't want to bloat it too much.

The check was swapped explicitly mainly to handle the "?!" at the end of a sentence, or someone saying something along the lines of `is your faceless void going battlefury? lul`. Happy to change it back, maybe even change it to a regex check if there is a ? in the last set of punctuation. `\w(?:\W*(\?)\W*)$`